http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/ProxyRoutes.scala

```diff
@@ -223,15 +223,6 @@ class ProxyRoutes(proxyService: ProxyService, corsSupport: CorsSupport, refererC
               .fromFuture(IO(request.entity.discardBytes().future))
         }
       resp <- ctx.span.fold(apiCall)(span => spanResource[IO](span, "proxyApp").use(_ => apiCall))
-
-      _ <- if (resp.status.isSuccess()) {
-        metrics.incrementCounter("proxyRequest",
-                                 tags = Map("result" -> "success", "action" -> "appRequest", "tool" -> s"${appName}"))
-      } else {
-        metrics.incrementCounter("proxyRequest",
-                                 tags = Map("result" -> "failure", "action" -> "appRequest", "tool" -> s"${appName}"))
```

Comment:
this `appName` is wrong...metrics are not aggregated properly this way. Hence did some refactoring to fix this